javascript/ql/src/Security/CWE-116/IncompleteHtmlAttributeSanitization.qhelp

```diff
@@ -38,6 +38,14 @@
 
 		</p>
 
+		<p>
+
+			An even safer alternative is to design the application
+			such that sanitization isn't needed at all, for instance by using HTML
```

Comment:
I'll leave it up to the documentation team to decide whether it's preferable to say "isn't" or "is not".